README.md

```diff
@@ -52,7 +52,10 @@ the details of your repo to produce a setting in the form:
 
 ```ruby
 REPO = 'https://YOUR_GITHUB_TOKEN@github.com/everypolitician-scrapers/kenya-mzalendo'
-ScrapedPageArchive.github_repo_url = REPO
+storage = ScrapedPageArchive::GitStorage.new
+storage.github_repo_url = REPO
```

Comment:
Is there a reason why this needs to be (able to be) set like this, rather than simply passed as an argument to `new()`? Constructors that don't take any arguments are a warnings sign.